nixos/modules/services/web-apps/mastodon.nix

```
@@ -190,12 +223,53 @@ in {
         type = lib.types.port;
         default = 55002;
       };
+
       sidekiqThreads = lib.mkOption {
-        description = lib.mdDoc "Worker threads used by the mastodon-sidekiq service.";
+        description = lib.mdDoc "Worker threads used by the mastodon-sidekiq-all service. If `sidekiqProcesses` is configured and any processes specify null `threads`, this value is used.";
```

```suggestion
        description = lib.mdDoc "Worker threads used by the mastodon-sidekiq-* services. If `sidekiqProcesses` is configured and any processes specify null `threads`, this value is used.";
```

This doesn't apply only to mastodon-sidekiq-all, right? Also could you put this line in a multi-line-string and wrap it?